uvicorn/config.py

```diff
@@ -180,7 +180,15 @@ def resolve_reload_patterns(
 
     directories = list(set(directories).difference(set(children)))
 
-    return (list(set(patterns)), directories)
+    return list(set(patterns)), directories
```

Comment:
if the return changes (I dont get why by the way) then why mypy doesnt catch this if you've not changed the signature ?

Reply:
@euri10 the return doesn't change. The two statements are exactly the same... the parentheses are simply redundant so I removed them (IntelliJ had a warning about it saying something like "remove redundant parentheses" so I did). It's like saying `if (True):` in python... it's exactly the same as `if True:` so there's no reason to write the parentheses.

Reply:
wahoo, my bad, totally misread that :face_with_spiral_eyes: